apps/api/src/app/shared/cache/cache-service.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
```

Comment:
the tests should be in the dal package I guess, there we don't use any testing library but you can introduce `jest` for ex., it provides great API to write the tests: https://jestjs.io/docs/getting-started
btw you can still write tests with `decribe, it` even if in the tests they use the `test` function

Reply:
Tent to agree regarding the location, in the initial i was not sure about couple of things but now it is clear :)
https://github.com/novuhq/novu/pull/2143/commits/f6f99528fac3d5c0bc48a20a5f5dc0fa87746bb6